docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+- Travel within the same location group requires two records in stop_times.txt with the same `location_group_id`.
+  - The first record with `pickup_type = 2` and `drop_off_type = 1` indicates that booking boarding is allowed at the location group.
+  - The second record with `pickup_type = 1` and `drop_off_type = 2` indicates that booking alighting is allowed at the location group.
+
+[**stop_times.txt**](../../reference/#stop_timestxt)
+
+trip_id | location_group_id | stop_sequence | start_pickup_drop_off_window | end_pickup_drop_off_window | pickup_type | drop_off_type | pickup_booking_rule_id | drop_off_booking_rule_id
+-- | -- | -- | -- | -- | -- | -- | -- | --
+476_weekdays | 476_stops | 1 | 17:30:00 | 22:00:00 | 2 | 1 | flächenrufbus_angermünde_weekdays | flächenrufbus_angermünde_weekdays
+476_weekdays | 476_stops | 2 | 17:30:00 | 22:00:00 | 1 | 2 | flächenrufbus_angermünde_weekdays | flächenrufbus_angermünde_weekdays
+476_weekends | 476_stops | 1 | 08:00:00 | 22:00:00 | 2 | 1 | flächenrufbus-angermünde_weekdays | flächenrufbus_angermünde_weekends
+476_weekends | 476_stops | 2 | 08:00:00 | 22:00:00 | 1 | 2 | flächenrufbus-angermünde_weekdays | flächenrufbus-angermünde_weekends
+
+## Deviated route
+
+"Route deviation" refers to services where the vehicle follows a fixed route with a set sequence of stops but has the flexibility to deviate from this route to pick up or drop off riders between stops. Typically, deviations are limited to keep the service on schedule, and advance booking is required for deviated pickups. 
+
+In this example, the [Hermann Express](https://www.newulmmn.gov/553/Hermann-Express-City-Bus-Service) service in New Ulm City allows users to board the service only at fixed stops and to be droped off at any point within a specific deviation area between these stops.
+
+**The example below has been simplified, download the [Hermann Express example dataset](../../../assets/deviated_drop-off_route.zip) for more details.**
+
+### Define trips
+
+Since this type of service still involves a series of fixed stops and a fixed schedule, defining trips is similar to normal fixed-route bus services. It requires defining trips for each service throughout the day.
+
+[**trips.txt**](../../reference/#tripstxt)
+
+route_id | service_id | trip_id | share_id
+-- | -- | -- | -- 
+74513 | c_67295_b_77497_d_31 | t_5374704_b_77497_tn_0 | p_1426044
+74513 | c_67295_b_77497_d_31 | t_5374699_b_77497_tn_0 | p_1426044
+74513 | c_67295_b_77497_d_31 | t_5374698_b_77497_tn_0 | p_1426044
+74513 | c_67295_b_77497_d_31 | t_5374697_b_77497_tn_0 | p_1426044
+... | ... | ... | ...
+
+### Define zones (GeoJSON location)
+
+Using locations.geojson to define zones for deviated route. Typically, deviations are limited to keep the service on schedule. Therefore, as the vehicle travels, the deviation area between each fixed stop may vary accordingly. The area for route deviation may look like the image below:
```

Comment:
When merged, add the link to the file

Reply:
Added link

Reply:
It doesn't seem to render properly on markdown 🤔.
<img width="457" alt="Screenshot 2024-04-05 at 1 39 47 PM" src="https://github.com/MobilityData/gtfs.org/assets/63653518/90adb9d5-5635-4ec1-ada7-01b761f9f1f8">

We're still missing links in a few other places where locations.geojson is referenced, and one place where booking_rules.txt is referenced.